myhpom/migrations/0006_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('myhpom', '0004_auto_20180708_0954'),
+        ('myhpom', '0005_health_networks'),
+    ]
```

Comment:
I don't believe this is a necessary migration: could just change the dependencies for the previous one. Since per the previous comment the migrations need to be split anyway if we start over by deleting these two migrations the newly-added one should have the right dependencies. The way to migrate your dev box back from these two so as to start over is to comment out the RunPytyhon so that the backwards migrate "./hstcl managepy migrate myhpom 0004" will work, then these migrations can be removed and replaced with two that first do schema alterations then load data in 2nd one.